chromium_src/chrome/browser/android/preferences/about_settings_bridge.cc

```diff
@@ -8,7 +8,7 @@
 
 #define JNI_AboutSettingsBridge_GetApplicationVersion \
   JNI_AboutSettingsBridge_GetApplicationVersion_ChromiumImpl
-#include "../../../../../../chrome/browser/android/preferences/about_settings_bridge.cc"
+#include "../../../../../chrome/browser/android/preferences/about_settings_bridge.cc"
```

Comment:
This looked correct before.

Reply:
hmm.. you're right. Not sure what happened there but I'll put it back